managed/services/supervisord/supervisord.go

```diff
@@ -425,8 +425,6 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 	clickhouseDatabase := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_DATABASE", defaultClickhouseDatabase)
 	clickhouseAddr := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_ADDR", defaultClickhouseAddr)
 	clickhouseDataSourceAddr := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_DATASOURCE_ADDR", defaultClickhouseDataSourceAddr)
-	clickhousePoolSize := getValueFromENV("PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE", "")
```

Comment:
Did they remove these two params from the client?

Reply:
OK, I see the commit message - it's no longer supported )

Reply:
In v2 pool size and block size changed. It is also recommended to set them agains exact DB. 

See: https://clickhouse.com/docs/en/integrations/go
MaxIdleConns should be alternative for pool size
BlockBufferSize for block size

But our block size default was 10 000 and BlockBufferSize did not accept such number, only lower ones, like 100. 

`time="2024-03-22T07:35:19.278+00:00" level=info msg="DSN: clickhouse://127.0.0.1:9000/pmm?block_buffer_size=10000&max_idle_conns=2" component=main
stdlog: Connection: strconv.ParseUint: parsing "10000": value out of range`

So seems it is slightly different. MaxIdleConns should behave same like pool size, but since both of them are with prefix "PERCONA_TEST" I decided to remove them.

Reply:
My fault. Thanks for notice that.

Reply:
Don't we treat variables prefixed with `PERCONA_TEST_` as non-public, internal ones?

I'm referring to https://github.com/percona/pmm/blob/70d5afdf8744bda5ab0563e868dfc2c063cd0aba/managed/CONTRIBUTING.md?plain=1#L54

Reply:
I mean they should be used in non-prod environments, i.e. test environments.

Reply:
You are right, seems like we can merge it to PMM 2 as well

Reply:
let's do it for v3 now, later we can backport to v2